.github/workflows/build.yml

```diff
@@ -3,12 +3,27 @@
 name: Build
 
 on:
-  workflow_dispatch
-
+  workflow_dispatch:
```

Comment:
It would be great to also have this workflow run automatically when a new tag is created, as done in `wazuh-kibana-app`

```yml
on:
  push:
    tags:
      - 'v*'
```

Reply:
For this approach, we need to take into account that the tag for the Security plugin should always be done before the tag for the Dashboard.

Reply:
Will discuss about it to possibly add it in a future PR